pkgs/development/python-modules/skorch/default.nix

```diff
@@ -2,28 +2,46 @@
 , stdenv
 , buildPythonPackage
 , fetchPypi
-, pytestCheckHook
-, flaky
+, pythonOlder
 , numpy
-, pandas
 , torch
 , scikit-learn
 , scipy
 , tabulate
 , tqdm
+, flaky
+, pandas
+, pytestCheckHook
+, safetensors
+, python
 }:
 
 buildPythonPackage rec {
   pname = "skorch";
-  version = "0.14.0";
+  version = "0.15.0";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-/d0s0N40W18uGfVbD9VEbhbWfduoo+TBqDjmTkjMUxs=";
+    hash = "sha256-39XVBlCmbg162z9uL84GZrU+v+M8waXbGdVV72ZYf84=";
   };
 
-  propagatedBuildInputs = [ numpy torch scikit-learn scipy tabulate tqdm ];
-  nativeCheckInputs = [ flaky pandas pytestCheckHook ];
+  disabled = pythonOlder "3.8";
+
+  propagatedBuildInputs = [
+    numpy
+    torch
+    scikit-learn
+    scipy
+    tabulate
+    tqdm
+  ];
```

Comment:
up to @bcdarwin but imho let's leave this as is to preserve the git blame. if we are going to be doing 💅 in this diff along with the upgrade, let's alphabetize

Reply:
I'm fine with this

Reply:
> if we are going to be doing 💅 in this diff along with the upgrade, let's alphabetize

done !